.release-please-manifest.json

```diff
@@ -1,4 +1,6 @@
 {
   "packages/runtime": "1.0.0",
-  "packages/next": "1.0.0"
+  "packages/next": "1.0.0",
```

Comment:
ah. the `packages/runtime` and `packages/next` shouldn't be there

Reply:
Yeah. See #42. 🙃